packages/flutter/lib/src/widgets/table.dart

```diff
@@ -339,6 +339,32 @@ class Table extends RenderObjectWidget {
   }
 }
 
+@immutable
+class _TableSlot {
+  const _TableSlot(this.x, this.y);
+
+  // The column that the child was in this table.
+  final int x;
+
+  // The row that the child was in this table.
+  final int y;
+
+  @override
+  bool operator ==(Object other) {
+    if (other.runtimeType != runtimeType)
+      return false;
+    return other is _TableSlot
+      && x == other.x
+      && y == other.y;
+  }
+
+  @override
+  int get hashCode => hashValues(x, y);
+
+  @override
+  String toString() => '${super.toString()}; "x" : "$x"; "y" : "$y"; }';
```

Comment:
see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#override-tostring for recommended boilerplate here

Reply:
Will fix this.

Reply:
Done.